README.md

```diff
@@ -168,6 +168,16 @@ This will fix linting errors such as double quotes and spacing issues. However,
    npm test
    ```
    After the tests have finished running, the results can be viewed in the console.
+   </details>
+
+<details>
+   <summary>About Errors</summary>
+   
+1. Prisma related errors are logged by th function logPrismaError with the specific error type.
+
+2.  Where applicable the error code is also logged. See the [Prisma documents] (https://www.prisma.io/docs/reference/api-reference/error-reference) for more information.
+
+3.  For more information about an error (such as stack trace) consider adding a log message specific to the error.
```

Comment:
This is not necessary to add to the README. The check box can be removed if it's not relevant. I put it there just as reminder if you made a change where you need to run certain commands for, or you need to install certain things for. The Docker PR is a good example for that or the test PR.

Reply:
Removed "About errors" section